### DISCUSSION TITLE
Issues with multiple instances of "calendar" module.

### DISCUSSION THREAD

**Discussion by morphout** [MMRIZE/MMM-CalendarExt3#159] [General]
Hi, I'm trying to run a calendar module using a calendar that is unique from the four calendars in my calendarExt3 module. Not all of the calendar .icals are loading into calendarExt3 from the hidden "calendar" module when I'm running a second instance of the "calendar" module to display the unique calendar events. Does anyone have any tips on getting the calendarExt3 to pull from four.ical calendars while showing a unique fifth calendar in a separate calendar module?

**Comment by eouia** [comment]
By default,  it should work. Maybe your calendar won't emit notification or something wrong. Give me more details.

---
Here is my example. I'm running two calendar modules and aggregate them into CX3.
![image](https://github.com/user-attachments/assets/cf9f31bb-db78-458c-9d93-1e86ba8afcae)

**Reply by ndanielh** [reply]
Eouia can i see the code in order to run 2 calendars?
